docs/js/getting-started.md

```diff
@@ -20,53 +20,296 @@ import LicenseBadge from '../../src/sap/sdk-js/LicenseBadge'
 <> <BuildBadge /> { } <LicenseBadge /> </>
 
 ## Installation
+
```

Comment:
Is this not good enough?

## CLI Usage

The CLI can initialize a nest-based project or (the more common case) add everything you need to develop for SAP Cloud Platform to an existing project no matter what backend framework you use.
If there are any incompatibilities, please let us know in the [issues](https://github.com/SAP/cloud-sdk-cli/issues/new/choose)!

Reply:
This is supposed to be the getting started section for the JS SDK. As a reader I now expect to be introduced in how to use the SDK, but the first sentence I read is - to install the CLI. Huh? SDK or CLI, what are we talking about?

I would rather expect to read something about the purpose of the SDK and that it can be used in applications on SAP Cloud Platform. To create such an application we provide a command line interface, that allows you to scaffold or enhance an application with the missing parts to use the SDK. And then we can say: 'To install the CLI...' because then the reader will know where we come from.